data/migrations/20200410153301_create_users_table.js

```diff
@@ -0,0 +1,7 @@
+exports.up = function(knex) {
+    
+  };
```

Comment:
Formatting on the closing of the function body is a bit off from where the function was declared, consider deleted to be on the same line as the declaration, or remove the extra tab.